docs/mining/proof-of-work.md

```diff
@@ -37,7 +37,7 @@ Pool mining distributes shares based on blocks found so you can earn a steady am
 
 ---
 
-## Obtain a Decred Address to Receive Funds With
+## ** Obtain a Decred Address to Receive Funds With **
```

Comment:
Nope just a small improvement in appearance and in making our use of markdown more correct/consistent. I will remove this if you wish